thunder/core/prims.py

```diff
@@ -2235,6 +2236,14 @@ def _exp2_number(a: Number) -> Number:
     output_dtype_kind=ELEMENTWISE_PRIM_OUTPUT_DTYPE_KIND.INT_FOR_NUMBER,
 )
 
+
+def frexp_meta(a: TensorProxy, /) -> (TensorProxy, TensorProxy):
+    utils.check_type(a, TensorProxy)
+    return TensorProxy(like=a), TensorProxy(like=a)
```

Comment:
I don't think this is quite right.  The second element here shouldn't have a dtype like `a`, but rather an integer dtype.  I think that's what's causing the `RuntimeError:  INTERNAL ASSERT FAILED at "/workspace/Fuser/csrc/runtime/executor_utils.cpp":491`

Reply:
Great catch!

Reply:
oops, my bad.

have updated the pr. Thanks!